files/sinonjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "sinon",
+  "repo": "sinonjs/sinon",
+  "files": {
+    "basePath": "lib/",
+    "include": ["sinon.js"]
```

Comment:
@jimaek Is it possible to set up auto updates on this project? I ask because as you said their NPM files don't look to be ready to setup in the browser. Also their GitHub releases don't have any files except a .zip file of the whole project and they don't release on Bower.